gallery_dl/extractor/aryion.py

```diff
@@ -126,7 +126,7 @@ def _parse_post(self, post_id):
             "user"  : self.user or artist,
             "title" : title,
             "artist": artist,
-            "path"  : text.split_html(extr("cookiecrumb'>", '</span'))[4:-1:2],
+            "path"  : text.split_html(text.unescape(extr("cookiecrumb'>", '</span')))[4:-1:2],
```

Comment:
```suggestion
            "path"  : text.split_html(text.unescape(extr(
                "cookiecrumb'>", '</span'))[4:-1:2],
```
to get `flake8` to stop complaining